### PR TITLE
Refactor dao apps

### PR DESCRIPTION
**Pull request by macor161**
# 🦅 Pull Request

Ref PR: #874

## 🚨 Test instructions

`dao apps`

### CODE REVIEW

File: packages/aragon-cli/src/commands/dao_cmds/apps.js [hunk 1]
```diff
@@ -51,6 +51,7 @@ exports.handler = async function({
 }) {
   knownApps = listApps(module ? [module.appName] : [])
   const web3 = await ensureWeb3(network)
+  let apps, daoAddress
```
Comment (0xGabi) [inline packages/aragon-cli/src/commands/dao_cmds/apps.js]: I change to use `ctx` to keep consistent with the rest of the codebase. We should discuss about best strategy on next sync

**Comment by 0xGabi** [comment]
As a reminder we should include integration tests here

**Comment by macor161** [comment]
This PR was a work in progress. I will post a link to the new completed PR this afternoon.

**Comment by 0xGabi** [comment]
Thanks @macor161 and sorry to merge it. I thought it was already done

**Pull request merged**